sw/device/lib/dif/meson.build

```diff
@@ -174,6 +174,21 @@ sw_lib_dif_pwrmgr = declare_dependency(
   )
 )
 
+# Key Manager DIF library
+# sw_lib_dif_keymgr = declare_dependency(
+#   link_with: static_library(
+#     'sw_lib_dif_keymgr',
+#     sources: [
+#       hw_ip_keymgr_reg_h,
+#       'dif_keymgr.c',
+#     ],
+#     dependencies: [
+#       sw_lib_mmio,
+#       sw_lib_bitfield,
+#     ],
+#   )
+# )
```

Comment:
Can you uncomment this so the header gets compiled as part of CI?

Reply:
Related to [this](https://github.com/lowRISC/opentitan/pull/4256#discussion_r526987407).